analytics/src/analytics/integrations/etldb/migrations/versions/2024_11_15_update_columns_etldb.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE IF NOT EXISTS gh_schema_version (
+	version INTEGER NOT NULL UNIQUE,
+	t_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	t_modified TIMESTAMP DEFAULT CURRENT_TIMESTAMP
+);
+
+INSERT INTO gh_schema_version (version) values (2) ON CONFLICT (version) DO NOTHING;
```

Comment:
This value (the schema version) is generated by looking at your position on the list of files, rather than being set here manually

Reply:
done